app/components/Hero.tsx

```diff
@@ -0,0 +1,43 @@
+// LIBRARY IMPORTS
+import { Box, Heading } from "@chakra-ui/react";
+import Image from "next/image";
+
+// LOCAL IMPORTS
+import heroImage from "../../public/images/hero-photo.png";
+
+// HERO COMPONENT DEFINITION
+export default function Hero() {
+  return (
+    <Box
+      position="relative"
+      textAlign="center"
+      bg="white"
+      minH="calc(38.25vw + 6rem)"
+    >
+      <Image
+        src={heroImage}
+        alt={"machines photo"}
+        layout="responsive"
+        width={1728}
+        height={610}
+        objectFit="cover"
+        objectPosition="center"
+      />
+      <Box
+        position="absolute"
+        w="93%"
+        left="50%"
+        transform="translate(-50%, -50%)"
+        height="6rem"
+        display="flex"
+        alignItems="center"
+        justifyContent="center"
+        opacity="90%"
+      >
+        <Heading size="xl" color="white">
+          Machining Services
```

Comment:
It seems like this font is defaulting to serif (the backup heading font) vs Roboto Slab. Refer to the screenshots in the PR. Anyone have ideas on why this might be?

Reply:
I wonder if we missed [this](https://chakra-ui.com/community/recipes/using-fonts) step when setting up fonts? I might be mistaken, but I think Brett set up fonts for Next.js, but there is also this additional step for ChakraUI if I understand correctly. I found the [same exact question](https://github.com/chakra-ui/chakra-ui/discussions/6386) in ChakraUI repo, but there is no answer :(

Reply:
@avinashi10 I think I found the answer, at least it works on my end now. I followed [instructions](https://stackoverflow.com/questions/75163876/reusing-next-font-globally-in-nextjs-13) for multiple fonts from Next.js docs, specifically creating a CSS variable for font and using those in ChakraUI theme because it looks like it doesn't pick up Roboto Slab on its own. 
1. I added `variable: "--font-roboto-slab"` and `variable: "--font-lato",` in `fonts.ts` file as suggested in Next.js docs.
2. In `layout.tsx`, I removed `className={lato.className}` from the body tag (line 20) and added ``className={`${lato.variable} ${roboto_slab.variable}`}`` to html tag (line 19)
3. In `theme.ts` I changed line 15 to `heading: "var(--font-roboto-slab)"`

Reply:
Yes, @alibacova has it right! I followed your steps and it works perfectly. Thank you for finding that fix!

Reply:
@alibacova that worked for me too, great find!

Reply:
works for me too!! great find!